ocp_resources/utils.py

```diff
@@ -218,3 +219,19 @@ def remaining_time(self):
         Return the remaining part of timeout since the object was created.
         """
         return self.start_time + self.timeout - time.time()
+
+
+def get_resource_timeout_sampler(resource, status, timeout, sleep):
+    LOGGER.info(
+        LOGGER.info(
+            f"Wait for {resource.kind} {resource.name} status to be {'ready' if status == True else status}"
```

Comment:
1. This is only applicable to VMs
2. Please remove the log back to where it was; this function retuns a sampler
3. There are more instances of this code in the repo; please re-use